content/docs/other-guides/istio-in-kubeflow.md

```diff
@@ -0,0 +1,96 @@
++++
+title = "Istio usage in Kubeflow"
+description = "Managing access to Kubeflow applications and resources via Istio"
+weight = 50
++++
+
+Kubeflow 0.6 onwards deploys Istio along with configuration to enable
+end-to-end authentication and access control. This setup is the foundation
+of multi-tenancy support in Kubeflow. A Kubeflow deployment without Istio is
+not possible.
+
+## A gentle introduction to Istio
+
+Most modern applications are built using a distributed microservices
+architecture. This ensures that each individual service is simple and has a
+well defined responsibility. Complex systems and platforms are generally
+built by composing many such microservices. Each microservice defines it's own
+APIs and the services interact with each other using these APIs in order to
+serve end-user requests.
+
+The term *service mesh* is used to describe the network of microservices that
+make up such applications and the interactions between them. As a service mesh
+grows in size and complexity, it can become harder to understand and manage.
+Its requirements can include discovery, load balancing, failure recovery,
+metrics, and monitoring. A service mesh also often has more complex operational
+requirements, like A/B testing, canary rollouts, rate limiting, access control,
+and end-to-end authentication.
+
+[Istio](https://istio.io/) is a pioneering and highly performant open-source
+implementation of service mesh by Google. Please consider reading through the
```

Comment:
Tech writer hint: The word "please" may be misunderstood in some cultures. It's clearer to be direct. Something like this:

```
For further details, you can read the [conceptual overview of Istio](https://istio.io/docs/concepts/what-is-istio/).
```

Reply:
reworded as suggested